.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: csharp
 solution: skalarprodukt.sln
 before_install:
-  - mkdir -p .nuget
-  - wget -O .nuget/nuget.exe https://dist.nuget.org/win-x86-commandline/v4.0.0/nuget.exe
-  - mono .nuget/nuget.exe # Output NuGet's version number
-install:
-  - mono .nuget/nuget.exe restore skalarprodukt.sln -Verbosity detailed
\ No newline at end of file
+  - mkdir ./.paket
+  - wget -O ./.paket/paket.bootstrapper.exe https://github.com/fsprojects/Paket/releases/download/3.18.2/paket.bootstrapper.exe
+  - mono ./.paket/paket.bootstrapper.exe 3.18.2
+  - mono ./.paket/paket.exe restore
+  - mono ./.paket/paket.exe add nuget FSharp.Compiler.Tools version 4.1.0
```

Comment:
~~I suggest that you do this command locally and commit the resulting `paket.dependencies`.~~

Aha, I got that. You want to install the package only on Travis, and conditionally add it into `fsproj` files. Alright then; that's okay I guess.